### PR TITLE
Fix for #4, #5, #6, #9

### PR DESCRIPTION
**Pull request by sbuser-git**
Resolved #XXXX.

### CODE REVIEW

File: cacti/plugins/benchmark/grid_benchmark_jobs.php
Comment (TheWitness) [inline cacti/plugins/benchmark/grid_benchmark_jobs.php]: This bit should be backed out.

**Review by TheWitness — CHANGES_REQUESTED** [review]
I believe that one line that I commented should be backed out, but the rest looks good.

File: cacti/plugins/benchmark/grid_benchmark_jobs.php
Comment (sbuser-git) [reply cacti/plugins/benchmark/grid_benchmark_jobs.php]: Thanks, will back it out.

**Comment by ddb4github** [comment]
1. #5 diff is OK for me,  suggest to rename funcname `stopRKey` because it is not related to "Retrun" key now.
2. Why change `$job_exitinfo` to `-1`,  function `getExceptionStatus` has avoid to display `TERM_UNKNOWN`.
3. #4 diff: How about use Cacti core function `check_changed` in diff of function `process_request_var`
4. #9 diff is OK for me
5. #6 diff is OK for me

**Comment by TheWitness** [comment]
@ddb4github, since the only thing that invalidates the "benchmark_id" is the "clusterid" change, I'm okay using that function as well.  It's your call.  You should be able to make the edits required to complete the merge right?

**Comment by sbuser-git** [comment]
Add 4 commits from Larry for: patch that disables the query cache for the Cacti Data Collectors - https://github.com/IBM/ibm-spectrum-lsf-rtm-server/issues/10

**Pull request merged**
